.azure/azure-pipelines.ci.yml

```
@@ -452,14 +452,13 @@ stages:
       platform: linux
       tls: stub
       extraArgs: -Filter -*TlsTest.CertificateError:TlsTest.ExtraCertificateValidation:*.Tcp*:*HandshakeParam*
-  # Disable macOS tests until datapaths are fixed
-  # - template: ./templates/run-bvt.yml
-  #   parameters:
-  #     image: macOS-10.15
-  #     platform: macos
-  #     tls: openssl
-  #     logProfile: None
-  #     extraArgs: -Filter -*DataECT0*:TlsTest.CertificateError:*.Tcp*:*DataPathTest.UdpData*:ParameterValidation.Validate*:*ParameterValidation/*:*Basic/WithFamilyArgs*:*AppData*:Misc.ServerDisconnect
+  - template: ./templates/run-bvt.yml
+     parameters:
+       image: macOS-10.15
+       platform: macos
+       tls: openssl
+       logProfile: None
+       extraArgs: -Filter -*.Tcp*
```

```suggestion
  - template: ./templates/run-bvt.yml
    parameters:
      image: macOS-10.15
      platform: macos
      tls: openssl
      logProfile: None
      extraArgs: -Filter -*.Tcp*
```
Looks like the spacing is off and AZP is complaining.